converters/toZigbee.js

```diff
@@ -586,6 +586,30 @@ const converters = {
             await entity.read('lightingColorCtrl', ['colorTemperature']);
         },
     },
+    light_colortemp_startup: {
+        key: ['color_temp_startup', 'color_temp_startup_percent'],
+        convertSet: async (entity, key, value, meta) => {
+            if (typeof value === 'string' && value.toLowerCase() == 'previous') {
+                // 0xffff = restore previous value
+                value = 65535;
+            } else if (key === 'color_temp_startup_percent') {
```

Comment:
remove `else` here and make it a separate if branch, otherwise `color_temp_startup_percent: 'previous'` won't work.

Reply:
How so? We check value will still be a string and set to previous right? So the first branch should be taken?

Reply:
Yes, I’d prefer to remove the _percent!